src/main/steps/c100-rebuild/miam/get-mediator/template.njk

```diff
@@ -0,0 +1,26 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
+<p class="govuk-body-l">{{paragraph}}</p>
+<p class="govuk-body-m">{{stepsLabel}}</p>
+
+<ol class="govuk-list govuk-list--number app-list--icon-steps ">
+    <li class="govuk-!-padding-bottom-6">
+    <form action="{{formActionUrl}}" target="_blank">
+        <input type="hidden" name="di" value="5">
+        <div class="govuk-form-group govuk-!-margin-bottom-3">
+            <label for="postcode-field" class="govuk-label">{{step1}}</label>
+            <input id="postcode-field" class="govuk-input govuk-!-width-one-third" autocomplete="postal-code" type="text" value="" name="pc">
+        </div>
+        <input type="submit" name="commit" value="Find a mediator" class="govuk-button govuk-!-margin-bottom-1" formnovalidate="formnovalidate">
```

Comment:
@kumarmadurai Can we move these values into content.ts & access here ?
e.g. 
```
defaultDistance: '5'
findMediatorButton: 'Find a mediator'
```

Reply:
Fixed